bamboo/compiler_tests/build_script.sh

```diff
@@ -1,4 +1,69 @@
+#!/bin/bash
+
 source /usr/share/lmod/lmod/init/bash
 source /etc/profile.d/00-modulepath.sh
+
 LBANN_DIR=$(git rev-parse --show-toplevel)
-${LBANN_DIR}/scripts/build_lbann_lc.sh --with-conduit
+CLUSTER=$(hostname | sed 's/[0-9]*//g')
+USER=$(whoami)
+WORKSPACE_DIR=$(ls --color=no -d /usr/workspace/*/${USER})
+DEPENDENCY_DIR=${WORKSPACE_DIR}/stable_dependencies/${CLUSTER}/gcc-7.3.0/mvapich2-2.3
+if [ -e ${DEPENDENCY_DIR} ];
+then
+    ml gcc/7.3.0 cmake/3.14.5 cuda
```

Comment:
Shouldn't this load a specific cuda since the following NCCL requires a particular one.

Reply:
Yes; I'll fix this when I update this script to work on other clusters (it only works on `pascal` right now), which, I assume, will be before CUDA 11 comes out.